htdocs/class/xoopsform/formtextdateselect.php

```diff
@@ -153,7 +153,11 @@ function showCalendar(id)
                 ');
             }
         }
-
-        return "<input type='text' name='" . $ele_name . "' id='" . $ele_name . "' size='" . $this->getSize() . "' maxlength='" . $this->getMaxlength() . "' value='" . $display_value . "'" . $this->getExtra() . " /><input type='reset' value=' ... ' onclick='return showCalendar(\"" . $ele_name . "\");'>";
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){            
+            return "<div class='form-inline'><input class='form-control' type='text' name='" . $ele_name . "' id='" . $ele_name . "' size='" . $this->getSize() . "' maxlength='" . $this->getMaxlength() . "' value='" . $display_value . "'" . $this->getExtra() . " /><input class='form-control' type='reset' value=' ... ' onclick='return showCalendar(\"" . $ele_name . "\");'></div>";
+        } else {
```

Comment:
__[phpmd]__
* The method render uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->